openverse_catalog/dags/common/requester.py

```diff
@@ -59,8 +62,11 @@ def get(self, url, params=None, **kwargs):
         """
         self._delay_processing()
         self._last_request = time.time()
+        request_kwargs = kwargs or dict()
+        if kwargs.get("headers") is None:
```

Comment:
This could be:
```suggestion
        if "headers" not in kwargs:
```

Reply:
It's not exactly the same, your suggestion is to check if the "headers" key is present in the dictionary but not if it has some _value_. The key might be present but still, be `None` for value.

Reply:
True. Are there times when someone might want to override the default to have no headers? If I stick with my version, people wouldn't have that option. I think I'll add a test case just to make it more clear.

Reply:
D'oh, you're both right! Sorry about that, ignore me!